.gitignore

```diff
@@ -44,7 +47,7 @@ app.*.map.json
 /android/app/debug
 /android/app/profile
 /android/app/release
-/packages/domain/pubspec.lock
-/packages/presentation/pubspec.lock
-/packages/data/pubspec.lock
-/pubspec.lock
+packages/presentation/pubspec.lock
+packages/data/pubspec.lock
+pubspec.lock
+packages/domain/pubspec.lock
```

Comment:
ignore all pubspec.lock in one line, like **/pubspec.lock

Reply:
ignored